### PR TITLE
don't call HC methods in methods (ahem)

### PR DESCRIPTION
**Pull request by cseed**
also includes full get_vcf_metadata.

### CODE REVIEW

File: python/hail/methods/impex.py [hunk 1]
```diff
@@ -598,23 +617,72 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
...
 @handle_py4j
 @typecheck(path=oneof(strlike, listof(strlike)))
 def get_vcf_metadata(path):
-    return Env.hc().get_vcf_metadata(path)
+    """Extract metadata from VCF header.
+
+    Examples
+    --------
+
+    >>> metadata = hc.get_vcf_metadata('data/example2.vcf.bgz')
```
Comment (tpoterba) [inline python/hail/methods/impex.py]: should be methods, not hc

File: python/hail/methods/impex.py [hunk 2]
```diff
@@ -598,23 +617,72 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
...
+
```
Comment (tpoterba) [inline python/hail/methods/impex.py]: can we actually print out the result here?

```
.. doctest::

    >>> metadata = methods.get_vcf_metadata...
    {'info' ...
```

File: python/hail/methods/impex.py [hunk 3]
```diff
@@ -598,23 +617,72 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
...
+    .. code-block:: text
+
+        ##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
+        ##FILTER=<ID=LowQual,Description="Low quality">
+        ##INFO=<ID=MQ,Number=1,Type=Float,Description="RMS Mapping Quality">
+
+    The resulting Python dictionary returned would be
+
+    .. code-block:: python
```
Comment (tpoterba) [inline python/hail/methods/impex.py]: pprint this:

```
{'filter': {'LowQual': {'Description': 'Low quality'}},
 'format': {'DP': {'Description': 'Read Depth',
                   'Number': '1',
                   'Type': 'Integer'}},
 'info': {'MQ': {'Description': 'RMS Mapping Quality',
                 'Number': '1',
                 'Type': 'Float'}}}
```

File: python/hail/methods/impex.py [hunk 4]
```diff
@@ -598,23 +617,72 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
...
+
+        metadata = {'format': {'DP': {'Number': '1', 'Type': 'Integer', 'Description': 'Read Depth'}},
+                    'filter': {'LowQual': {'Description': 'Low quality'}},
+                    'info': {'MQ': {'Number': '1', 'Type': 'Float', 'Description': 'RMS Mapping Quality'}}}
+
+    which can be used with :meth:`.export_vcf` to fill in the relevant fields in the header.
+    
+    Parameters
+    ----------
+    path : str or list of str
```
Comment (tpoterba) [inline python/hail/methods/impex.py]: kinda like using the 
```
:obj:`str` or :obj:`list` of :obj:`str`
```
for how it renders. Maybe it's too hard to read as plain text? I dunno

Comment (tpoterba) [reply python/hail/methods/impex.py]: why does this read multiple files? That seems like the wrong design. It should just take one. Can we fix that here, or should we do in a separate PR?

Comment (cseed) [reply python/hail/methods/impex.py]: It just parses the first one.  Will fix.

Comment (cseed) [reply python/hail/methods/impex.py]: OK, fixed.

**Pull request merged**